docs/finetune/anyscale.mdx

```diff
@@ -35,11 +36,11 @@ USING
 Then, create a model using this engine:
 
 ```sql
-CREATE MODEL myllama2
+CREATE MODEL mymistral7b
 PREDICT completion
 USING
     engine = 'anyscale_engine',
-    model_name = 'meta-llama/Llama-2-7b-chat-hf',
+    model_name = 'mistralai/Mistral-7B-Instruct-v0.1',
     prompt_template = 'Return a valid SQL string for the following question about MindsDB in-database machine learning: {{prompt}}';
 ```
 
```

Comment:
Let's update the model name to `mymistral7b` in the below DESCRIBE command.

Reply:
Done, good catch 😄